internal/pkg/cli/app_delete.go

```diff
@@ -314,6 +315,14 @@ func (o *deleteAppOpts) emptyS3Bucket() error {
 	}
 	appResources, err := o.cfn.GetRegionalAppResources(app)
 	if err != nil {
+		innerErr := err
+		for errors.Unwrap(innerErr) != nil {
+			innerErr = errors.Unwrap(innerErr)
+		}
+		// ErrStackSetNotFound means application Parameter Store params are hanging, return nil to continue for deletion
+		if _, ok := innerErr.(*stackset.ErrStackSetNotFound); ok {
+			return nil
+		}
```

Comment:
Can we reuse `isStackSetNotExistsErr`? Also we can potentially simplify `isStackSetNotExistsErr` because `Unwrap` is already implemented in a recursive way.

Reply:
Reading the docs and testing, I don't think `Unwrap` is recursive at all 🤔

Reply:
i think @iamhopaul123 meant [`errors.Is()`](https://pkg.go.dev/errors#Is), which recursively calls unwrap until it finds the target error (or `errors.As()`, which is _probably_ what you want)